pages/about.tsx

```diff
@@ -0,0 +1,74 @@
+import Layout from "@components/layout";
+import { ReactNode } from "react";
+import styles from "styles/pages/About.module.scss";
+
+interface IAbout {
+  title: string;
+  value: string | ReactNode;
+}
+
+const aboutContainers: IAbout[] = [
+  {
+    title: "Team History",
+    value: `We were founded in 2010 (yes, we're ${
+      new Date().getFullYear() - 2010
+    } years old) and we have competed in the FRC Robotics Competition every year since. We operate out of Mounds View High School and the majority of our students attend there as well.`,
+  },
+  {
+    title: "Our Mission",
+    value:
+      "Are main goal is to improve the skills of our team members (Mentors and Students) to help them be more successful in their current and future endeavers in STEM.",
+  },
+  {
+    title: "Key Team Values",
+    value: (
+      <ul style={{ textAlign: "left" }}>
+        <li>It{"'s"} not stupid if it works</li>
```

Comment:
I think we have a different set of values somewhere, actually.  So perhaps comment this out until we have the right set.

Reply:
Fixed